packages/plugin-credentials/src/credentials/common.js

```diff
@@ -150,6 +154,10 @@ export default {
   getUserToken(scope) {
     // Note: this behaves much like oneFlight, but doesn't return a unique
     // promise. Since it recursively calls iteself, oneFlight is problematic.
+    if (this.isLoggingOut) {
+      return Promise.reject(`credentials: Cannot get UserToken while logging out`);
```

Comment:
`credentials: Credentials#getUserToken() is not available while Credentials#isLoggingOut is true`